egs2/libritts/speechlm1/conf/train_multiscale.yaml

```diff
@@ -0,0 +1,62 @@
+# Model
+model: espnet
+model_conf:
+    share_emb: false
+  
+corelm: espnet_builtin
+corelm_conf:
+    pos_enc: sinusoidal
+    att_unit: 512
+    head: 8
+    unit: 1024
+    encoder_layer: 8
+    decoder_layer: 8
+    dropout_rate: 0.0
+    positional_dropout_rate: 0.0
+    attention_dropout_rate: 0.0
+    flashattention: true
+  
+predictor: multiscale
+predictor_conf:
+    pos_enc: sinusoidal
+    att_unit: 256
+    head: 4
+    unit: 512
+    decoder_layer: 1
+    dropout_rate: 0.0
+    positional_dropout_rate: 0.0
+    attention_dropout_rate: 0.0
+    flashattention: true
+
+postprocessor: none
+
+# Dataloader
+batch_type: numel
+batch_bins: 50000
```

Comment:
Seems to be very small, is it for test purpose?

Reply:
It means the number of tokens (mainly frames)